packages/fe-mockserver-plugin-cds/src/index.ts

```diff
@@ -26,6 +27,29 @@ export default class CDSMetadataProvider implements IMetadataProcessor {
      */
     constructor(private fileLoader: IFileLoader, private options?: CDSMetadataProviderOptions) {}
 
+    async loadI18nMapFolder(targetFolder: string, i18nMap: Record<string, string>): Promise<void> {
+        if (await this.fileLoader.exists(targetFolder)) {
+            const i18nProp = await this.fileLoader.loadFile(targetFolder + 'i18n.properties');
+            const i18nPropLines = i18nProp.split('\n');
```

Comment:
leads to somewhat different results if the .properties file uses CRLF line endings... (the value keeps the `\r`) - did not check if that is a problem / whether trimming would be an option?

Reply:
trimming would help i guess :)